tests/app/parser/test_metadata_parser_v2.py

```diff
@@ -0,0 +1,201 @@
+from copy import deepcopy
```

Comment:
Are these not identical to v1 bar the fixture?. If so, parametrize?

Reply:
Updated 👍